queries.yaml

```diff
@@ -728,6 +730,7 @@ post_process:
         - disputed_by
         - "disputed_by:left"
         - "disputed_by:right"
+        - recognized_by
```

Comment:
Do we need `"recognized_by:left"` and `"recognized_by:right"`, too?

Reply:
I don't think we create them but it won't hurt to add to the list.  Also going to add tz_admin_level because it doesn't appear to be getting dropped

Reply:
fixed in 9506b97782500824588c5745c04dc520b202da17